python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -236,7 +236,8 @@ def _get_current_usage(self, topology: Topology) -> ExecutionResources:
             if isinstance(op, InputDataBuffer):
                 continue  # Don't count input refs towards dynamic memory usage.
             for bundle in state.outqueue:
-                cur_usage.object_store_memory += bundle.size_bytes()
+                if cur_usage.object_store_memory:
```

Comment:
This may undercount memory. We should initialize all resource fields to 0 in cur_usage instead.

Reply:
Done.